packages/palette/src/elements/Clickable/Clickable.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react"
+import styled from "styled-components"
+import { boxMixin, BoxProps } from "../Box"
+
+/**
+ * ClickableProps
+ */
+export type ClickableProps = React.ButtonHTMLAttributes<HTMLButtonElement> &
+  BoxProps
+
+/**
+ * Clickable is a utility component useful for wrapping things like <div>s
+ * without having to deal with the requirements to make the <div> accessible.
+ */
+export const Clickable = styled.button<ClickableProps>`
```

Comment:
might want to import `styled` from our [primitives](https://github.com/artsy/palette/blob/master/packages/palette/src/platform/primitives.ts#L16) as this component can be cross-platform perhaps.

Reply:
I could also just alias `TouchableWithoutFeedback` which should basically be the same thing?

Reply:
Re primitives, it's just a pattern where where import from there because thats how our universal component system works. We haven't been diligent with this in the past and its led to errors in Eigen, but it's also admittedly hard to remember.